maplibre/src/map_state.rs

```diff
@@ -1,3 +1,5 @@
+//! Stores the state of the map such as zoom level, camera, styles, etc.
```

Comment:
```suggestion
//! Stores the state of the map such as [`Zoom`], camera, styles, etc.
```

Maybe we should link here the proper structs.